lib/ansible/modules/cloud/vmware/vsphere_guest.py

```diff
@@ -986,11 +920,11 @@ def reconfigure_vm(vsphere_client, vm, module, esxi, resource_pool, cluster_name
     if vm_disk:
         spec = spec_singleton(spec, request, vm)
 
-        # Get a list of the VM's hard drives
-        dev_list = [d for d in vm.properties.config.hardware.device if d._type=='VirtualDisk']
+        # Get a list of the VMs hard drives
+        dev_list = [d for d in vm.properties.config.hardware.device if d._type == 'VirtualDisk']
         if len(vm_disk) > len(dev_list):
             vsphere_client.disconnect()
-            module.fail_json(msg="Error in vm_disk definition. Too many disks defined in comparison to the VM's disk profile.")
+            module.fail_json(msg="Error in vm_disk definition. Too many disks defined in comparison to the VMs disk profile.")
```

Comment:
this was correct before IMHO

Reply:
Hah, I looked it up and in fact both are perfectly fine. I always felt that writing VM's was more Dutch than English, because you would rather write CDs in English, rather than CD's. It seems both are fine, although not using the apostrophe seems to be preferred (less awkward).